lib/phonegap/cordova-dependence.js

```diff
@@ -69,9 +72,25 @@ module.exports.exec = function(projectDir, extEvents) {
                 var cordovaCommand = 'npm install cordova --save';
                 extEvents.emit('log', 'Adding Cordova dependency with: "' + cordovaCommand + '"');
                 extEvents.emit('log', 'This may take a few minutes ...');
+
+                var spinner = ora('installing cordova').start();
+
+                var startTime = Date.now();
+                analytics.trackEvent([], 0, {
+                    short_message: "npm-install-cordova-start",
+                    _params: []
+                });
+
                 var child = shell.exec(cordovaCommand, {
                     silent: true
                 }, function(code, stdout, stderr) {
+                    spinner.stop();
+                    var dT = Date.now() - startTime;
+                    var dTsec = Math.round(dT / 1000) + "s"; // analytics does not like numbers,
+                    analytics.trackEvent([dTsec], code, {
+                        short_message: "npm-install-cordova-finish",
+                        _params: [dTsec]
```

Comment:
I would create a separate property on this object that represents the duration. `_params` is used for command parameters passed to the phonegap-cli, so I'd rather we keep that consistent.

How about `_duration: dtSec`? It would be easier to, down the road, query the duration directly in our analytics, rather than having to deal with the wrapper `_params` array. With the current implementation, future analytics ElasticSearch queries for, for example, cordova installs that take longer than 1000ms would look like `short_message:"npm-install-cordova-finish" AND params[0] > 1000`. If you change that property to `_duration`, it would be `short_message:"npm-install-cordova-finish" AND duration > 1000` - perhaps a bit more intuitive.